### PR TITLE
Maintainer Status

### PR DESCRIPTION
**Pull request by thebioengineer**
Add Ellis as maintainer. Add comment to Gregs section and remove maintainer status. Add Jimmy, Marie, and Jason as contributors

## Description

- Added Ellis as the maintainer of the Package
- Removed maintainer status for Greg, but preserved ctb and cph.
- Added comment for Greg indicating original author and contributor
- Added Jimmy, Marie, and Jason as contributors

### CODE REVIEW

**Review by mariev — COMMENTED** [review]
1. I am wondering what the plan is for dealing with CRAN expectations for handoff? 
2. Should we be using personal emails for FHCRC/SCHARP staff?

**Review by jmtaylor-fhcrc — APPROVED** [review]

**Comment by thebioengineer** [comment]
I've used my personal email for past CRAN submissions, so I was doing it for consistency. 

@gfinak - would you like the email listed for you to be updated to the one you list on your github page now?

**Comment by gfinak** [comment]
@thebioengineer just saw this. You can set my email to greg.finak@gmail.com

File: DESCRIPTION [hunk 1]
```diff
@@ -2,9 +2,15 @@ Package: DataPackageR
 Type: Package
 Title: Construct Reproducible Analytic Data Sets as R Packages
 Authors@R:
-    c(person(given = "Greg Finak", role=c("aut","cre","cph"), email="gfinak@fredhutch.org"),
+    c(person(given = "Greg Finak", 
+             role=c("aut","cph"),
+             email="gfinak@fredhutch.org",
+             comment="Original author and creator of DataPackageR"),
```
Comment (wfulp) [inline DESCRIPTION:8]:
```suggestion
             email="greg.finak@gmail.com",
             comment="Original author and creator of DataPackageR"),
```

File: DESCRIPTION [hunk 2]
```diff
@@ -2,9 +2,15 @@ Package: DataPackageR
...
       person(given = "Paul Obrecht", role=c("ctb")),
-      person(given = "Ellis Hughes", role=c("ctb")),
+      person(given = "Ellis Hughes", role=c("ctb","cre"), email = "ellishughes@live.com"),
+      person(given = "Jimmy Fulp", role=c("ctb")),
```
Comment (wfulp) [inline DESCRIPTION:11]:
```suggestion
      person(given = "Jimmy Fulp", role=c("ctb"), email = "william.fulp@gmail.com"),
```

**Review by wfulp — APPROVED** [review]
Update Greg and my emails

**Review by mariev — APPROVED** [review]

**Pull request merged**